### PR TITLE
First example test: sed file system

### PR DESCRIPTION
**Pull request by EECOLOR**

### CODE REVIEW

**Comment by EECOLOR** [comment]
Again, be aware this is based on `refactoring_opts`.

File: suffuse/examples_test.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    // seq.txt
+    // 10 seq.txt
+    // 4
+    // 5
+    // 6
+    // -rw-rw-r-- 1 13 Jul 10  2015 seq.txt#5,10p
+    // -rw-rw-r-- 1 6 Jul 10  2015 seq.txt#1,3p
+
+  })
+}
```
Comment (paulp) [inline suffuse/examples_test.go:41]: This looks awesome given what's possible inside go's syntactic limitations, but given that this is really nothing but shelling out, does it offer some benefit over something along the lines of 

``` go
executeBash(`
  ls                      # List the contents through the suffuse mount.
  wc -l seq.txt           # It's a 10 line file, one number to a line.
  cat 'seq.txt#4,6p'      # Via suffuse, a derived file ending with #4,6p is...
  ls -gG 'seq.txt#5,10p'  # Arbitrary sed commands, different sized files.
  ls -gG 'seq.txt#1,3p'   # These files are effectively indistinguishable from "real" files.
`)
```

AFAIK go ships with no string interpolation. I'm sure there are a bunch of named string interpolation libraries which plus-or-minus call "ReplaceAll" and call it a day. Anyway, with or without interpolation I feel like the direct expression of these commands is superior (even if you want to keep all the whitespace.)

Comment (EECOLOR) [inline suffuse/examples_test.go:41]: Good points, and I agree.

**Comment by EECOLOR** [comment]
@paulp any idea how I can set up the file rights so they match on my local machine and the travis server (see problematic travis build)?

**Comment by paulp** [comment]
At the moment suffuse is semi-arbitrarily setting files perms to 0644 and dirs to 0755. That needs to be improved. Regardless, you can get the same result at home by setting your umask to 022.

```
% umask 002
% touch file.txt
% ls -l file.txt
-rw-rw-r--  1 paulp  wheel  0 Jul  8 09:49 file.txt
% umask 022
% touch file2.txt
% ls -l file2.txt
-rw-r--r--  1 paulp  wheel  0 Jul  8 09:49 file2.txt
%
```

**Comment by EECOLOR** [comment]
I still need to make the other suggested change

**Comment by EECOLOR** [comment]
I think this indeed dramatically improves the situation.

**Comment by paulp** [comment]
Oh, great idea with the multiple argument lists. This is way better.

**Comment by paulp** [comment]
Oops, thought I was merging that into master. My bad.

**Comment by EECOLOR** [comment]
Yeah, it requires the refactoring of opts. I don't mind it being merged inthere

**Pull request merged**